indico/modules/events/layout/__init__.py

```diff
@@ -37,7 +40,9 @@
     'timetable_theme_settings': {},
     'use_custom_menu': False,
     'timetable_by_room': False,
-    'timetable_detailed': False,
```

Comment:
why remove that comma? ;x

Reply:
Why not? 😉

Reply:
unrelated to the other changes and no real benefit in removing it :p

Reply:
Whoops, it wasn't intentional

Reply:
This is why we need Black.

Reply:
I still don't like it very much because of their weird decision on quote style, but without quote normalization and 120-char line length I think it wouldn't really change a lot for us.